File: python/pizza_party.py
number_of_people = input("How many people? ")
try:
    people = int(number_of_people)
except ValueError:
    print("Please enter a valid number!!")
    exit(0)

number_of_pizzas = input("How many pizzas do you have? ")
try:
    pizzas = int(number_of_pizzas)
except ValueError:
    print("Please enter a valid number!!")
    exit(0)

leftovers = people % pizzas
print(f"{people} people with {pizzas} pizzas")
print(f"Each person gets {(people / pizzas) / 2} pieces of pizza.")
print(f"There are {leftovers} leftovers pieces.")

File: c/pizza_party.c
#include <stdio.h>
#include <stdlib.h>

int main(void) {
  int people = 0;

  printf("How many people? ");
  scanf("%d", &people);

  if (people <= 0) {
    printf("Please enter a valid number!!\n");
    exit(1);
  }

  int pizzas = 0;
  printf("How many pizzas do you have? ");
  scanf("%d", &pizzas);
  if (pizzas <= 0) {
    printf("Please enter a valid number!!\n");
    exit(1);
  }

  int leftovers = people % pizzas;
  printf("%d people with %d pizzas\nEach person gets %d pieces of pizza.\n",
         people, pizzas, (people / pizzas) / 2);
  printf("There are %d leftovers pieces.\n", leftovers);
}
